pyproject.toml

```
@@ -23,7 +23,8 @@ classifiers = [
 
 dependencies = [
     "requests==2.32.3",
-    "click>=8.1"
+    "click>=8.1",
```

Click is already a dep of the toolkit. But it is fine to leave here.
I would pin to the point releases, it is more reproducible

For example
```suggestion
    "click==8.1",
```